packages/angular/src/app/pages/planning-task-list/task-list-gantt/task-list-gantt.component.ts

```diff
@@ -0,0 +1,42 @@
+import {
+  Component, NgModule, Input, SimpleChanges, OnChanges,
+} from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { HttpClientModule } from '@angular/common/http';
+import { Subscription } from 'rxjs';
+import {
+  DxGanttModule,
+  DxLoadPanelModule,
+} from 'devextreme-angular';
```

Comment:
Use long pah to module to improve tree shaking (devextreme-angular/ui/gantt)

Reply:
Fixed